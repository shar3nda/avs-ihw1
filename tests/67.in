10
2 6 6 5 5 4 1 4 5 4