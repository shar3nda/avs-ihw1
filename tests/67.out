-1 6 7 -1 1 4 3 8 3 4 2 0 6